makefile

```diff
@@ -0,0 +1,35 @@
+$(shell mkdir -p bin)
+$(shell mkdir -p build)
+
+CC=g++
+CFLAGS=-std=c++17
+
+INCLUDECADMIUM=-I ../../lib/cadmium
```

Comment:
correct the path to include **cadmium header files** from the **lib** project folder.

Reply:
suggested changes in  the paths has been made, please review  he request again

Reply:
issue resolved--verified